content/snapshots.md

```diff
@@ -10,19 +10,19 @@ Take a disk snapshot of a persistent disk before deploying major updates or for
 
 ## Enabling Snapshots {: #enable }
 
-Since the IaaS might or might not provide snapshotting functionality, disk snapshots are disabled by default. If your IaaS supports snapshots, you must enable snapshots in your IaaS and in the Director to use disk snapshots.
+Since the IaaS might or might not provide snapshotting functionality, disk snapshots are disabled by default. If your IaaS supports snapshots, you must first enable snapshots in your IaaS and then in the Director.
 
-To enable disk snapshots:
+To enable disk snapshots in the Director:
 
-1. Change the deployment manifest for the Director:
+1. Add an `enable_snapshots` key with it's value set to `true` to the `director` block of your Director deployment manifest.
 
     ```yaml
     properties:
       director:
         enable_snapshots: true
     ```
 
-1. Run `bosh deploy` to update your deployment.
+1. Run `bosh deploy` to update your Director deployment.
```

Comment:
it looks like this was wrong originally. It would be a `create-env`, typically, to update the director itself. `bosh deploy` is for deploying jobs/releases with the director.

Reply:
You're right. I fixed the command, andd added links to command and key details.